packages/plugins/users-permissions/admin/src/pages/Providers/index.js

```diff
@@ -204,9 +196,6 @@ export const ProvidersPage = () => {
                       condition: canUpdate,
                     })}
                   >
-                    <Td width="">
-                      <FontAwesomeIcon icon={provider.icon} />
```

Comment:
How will this effect the SSO providers if at all? 🤔

Reply:
I think it won't (will check that tomorrow). I've discussed it with Maeva and she agreed that we can remove the icon from the table view in the admin UI and I don't think it is used anywhere else.

Reply:
Good question! The SSO documentation mentions an `icon` attribute. According to the [documentation it should hold a URL](https://docs.strapi.io/developer-docs/latest/setup-deployment-guides/configurations/optional/sso.html) pointing to a logo. We should be safe here 🤞🏼